build_and_push.sh

```diff
@@ -27,10 +27,42 @@ docker build \
     .
 
 # verify
-docker run "$TAGGED_IMAGE" sh -c "starknet --version \
+# verify and assert contract compilation
+cairo_0_contract="contracts/cairo0/contract.cairo"
+cairo_1_contract="contracts/cairo1/contract1.cairo"
+
+dir="$(pwd)"
+cairo_1_output="$dir/artifacts/contracts/cairo1"
+cairo_0_output="$dir/artifacts/contracts/cairo"
+
+mkdir -p "$cairo_1_output"
+mkdir -p "$cairo_0_output"
+
+docker run -v "$dir":"$dir" "$TAGGED_IMAGE" sh -c "starknet --version \
     && starknet-compile-deprecated --version \
     && /usr/local/bin/target/release/starknet-compile --version \
-    && /usr/local/bin/target/release/starknet-sierra-compile --version"
-# push
-docker push "$TAGGED_IMAGE"
-docker push "$LATEST_IMAGE"
+    && /usr/local/bin/target/release/starknet-sierra-compile --version \
```

Comment:
No need to stack all of this into a single `docker run` command. Why not separate cairo0 and cairo1 testing (at least into two functions of this file)?

Reply:
https://github.com/0xSpaceShard/cairo-cli-docker/pull/18/commits/c0b97d630d63ce2722d8753674165bd184070b9f